src/Core/Framework/DataAbstractionLayer/FieldSerializer/PriceDefinitionFieldSerializer.php

```diff
@@ -202,6 +202,12 @@ private function validateRules(array $data, string $basePath): ConstraintViolati
             if (!$rule instanceof Container) {
                 $rule->assign($data);
                 $validations = $rule->getConstraints();
+                $config = $rule->getVars() ?? [];
+                if(is_array($config) && array_key_exists('operator', $config) && array_key_exists('amount', $config)) {
```

Comment:
Also a Unit Test would be nice to verify this change

Reply:
Hey, while I was writing a test, I saw that the error is gone on SW6.4.17.1.
Before you delete this pull request, do you know maybe where is the change?
It is not mentioned in the changelog, but it would useful for shops that cannot update yet to the newest version.

Reply:
Oh, I think I see change in src/Core/Framework/Rule/Rule.php, that could be it